### PR TITLE
Fix setup for Egress Gateway with NetworkPolicy doc

### PR DESCRIPTION
**Pull request by rlenglet**
Fixes: https://github.com/istio/istio/issues/14736

### CODE REVIEW

File: content/docs/tasks/traffic-management/egress/egress-gateway/index.md [hunk 1]
```diff
@@ -650,6 +650,58 @@ external service.
     sleep istio-proxy
     {{< /text >}}
 
+1.  Create the same destination rule as for the `sleep` pod in the `default` namespace to direct the traffic through the egress gateway:
```
Comment (frankbu) [inline content/docs/tasks/traffic-management/egress/egress-gateway/index.md]: What is this for? This exact same destination rule is already created in step 1.

Comment (rlenglet) [reply content/docs/tasks/traffic-management/egress/egress-gateway/index.md]: No, it's created in a different namespace (`default` vs. `test-egress`).

Comment (rlenglet) [reply content/docs/tasks/traffic-management/egress/egress-gateway/index.md]: Doh I need to fix the command to add `-n test-egress` as I verified fixed the issue.

**Comment by geeknoid** [comment]
Can you please rebase and repush, which will hopefully fix the CI problems? 

Thanks.

**Comment by rlenglet** [comment]
> Can you please rebase and repush, which will hopefully fix the CI problems?

Done.

**Pull request merged**